lib/gzr/commands/attribute.rb

```diff
@@ -101,7 +101,7 @@ def import(file)
       method_option :'can-edit', type: :boolean, default: true,
                            desc: 'can a user change the value themself'
       method_option :'domain-whitelist', type: :string,
```

Comment:
any chance we can change or alias this to `domain-allowlist`?

Reply:
done